platforms/android/android.toolchain.cmake

```diff
@@ -711,6 +712,14 @@ elseif( ANDROID_ABI STREQUAL "armeabi-v7a with NEON" )
  set( CMAKE_SYSTEM_PROCESSOR "armv7-a" )
  set( VFPV3 true )
  set( NEON true )
+elseif( ANDROID_ABI STREQUAL "armeabi-v7a-hard with NEON" )
+ set( ARMEABI_V7A_HARD true )
+ set( ANDROID_NDK_ABI_NAME "armeabi-v7a-hard" )
```

Comment:
Result libraries (OpenCV and examples) will be put into '/lib/armeabi-v7a-hard` and will not be found in the apk, installation paths should be updated. Probably, new variable is needed to store default installation path.